src/mame/heathkit/tlb.cpp

```diff
@@ -212,6 +212,17 @@ void heath_tlb_device::device_start()
 
 	m_key_click_timer = timer_alloc(FUNC(heath_tlb_device::key_click_off), this);
 	m_bell_timer = timer_alloc(FUNC(heath_tlb_device::bell_off), this);
+
+	// flip bits in font ROM to avoid having to flip them during scan out since some
+	// boards with graphics has the pixel graphic reversed from the font layout. Doing
+	// it for all devices for consistency.
+	memory_region *region = device().memregion("chargen");
+	int font_rom_size = (region != nullptr)? region->bytes() : 0;
+
+	for(int i = 0; i < font_rom_size; i++)
```

Comment:
Since `m_p_chargen` is a `required_region_ptr`, you can’t get here if the memory region doesn’t exist.  You can also call `.length()` on it to get the length in elements, and you don’t need to call `.device()` to get `device_t` members in a `device_t` implementation.